<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>790</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>759</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IntelPy</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/pieico/goodpie2.ico</normaloff>:/pieico/goodpie2.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="mainHorizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="leftSideVerticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QTextEdit" name="logTextEdit">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearLogButton">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Clear Log</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightSideVLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tab_widget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>370</width>
            <height>0</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="main_tab">
           <attribute name="title">
            <string>Main</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>361</width>
              <height>51</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="labelHome_System">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Home system:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditHome_System">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButtonSet_Home">
               <property name="text">
                <string>Set Home</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>80</y>
              <width>361</width>
              <height>79</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="alertLayout">
             <item>
              <widget class="QLabel" name="labelAlertWhen">
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Alert when within jumps:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_AlertJumps">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>5</number>
               </property>
               <property name="pageStep">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_alertjumps">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>160</y>
              <width>361</width>
              <height>551</height>
             </rect>
            </property>
            <property name="title">
             <string>Recent Alerts</string>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget_3">
             <property name="geometry">
              <rect>
               <x>9</x>
               <y>35</y>
               <width>341</width>
               <height>352</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_recentalert1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Alert1</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_recentalert2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Alert2</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_recentalert3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Alert3</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_recentalert4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Alert4</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_recentalert5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Alert5</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="advancedtab">
           <attribute name="title">
            <string>Config</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>361</width>
              <height>111</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="labelEveLogLocation_2">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Eve logs:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButtonChoose_Eve_Log_Location">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Choose...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Alarm:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButtonChoose_alarm_Location">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Choose...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_alarm">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditEve_Log_Location">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>520</y>
              <width>192</width>
              <height>92</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="checkbox_displayalerts">
               <property name="text">
                <string>display alert details</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_displayclear">
               <property name="text">
                <string>display clear messages</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_displayall">
               <property name="text">
                <string>display all messages</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>520</y>
              <width>91</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>Log options:</string>
            </property>
           </widget>
           <widget class="QWidget" name="verticalLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>630</y>
              <width>197</width>
              <height>61</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="checkBox_filterclear">
               <property name="text">
                <string>ignore clear messages</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_filterstatus">
               <property name="text">
                <string>ignore status messages</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>630</y>
              <width>101</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>Filter options:</string>
            </property>
           </widget>
           <widget class="QGroupBox" name="watchedChannelsGroupBox">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>120</y>
              <width>361</width>
              <height>330</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>330</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>330</height>
             </size>
            </property>
            <property name="title">
             <string>Watched channels</string>
            </property>
            <widget class="QListWidget" name="channelListWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>80</y>
               <width>341</width>
               <height>200</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
             <property name="selectionRectVisible">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="removeChannelButton">
             <property name="geometry">
              <rect>
               <x>210</x>
               <y>280</y>
               <width>141</width>
               <height>34</height>
              </rect>
             </property>
             <property name="text">
              <string>Remove Channel</string>
             </property>
            </widget>
            <widget class="QPushButton" name="addChannelAddButton">
             <property name="geometry">
              <rect>
               <x>260</x>
               <y>40</y>
               <width>91</width>
               <height>34</height>
              </rect>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
            <widget class="QLabel" name="labeldotimperium">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>50</y>
               <width>81</width>
               <height>22</height>
              </rect>
             </property>
             <property name="text">
              <string>.imperium</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEditChannelAdd">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>161</width>
               <height>36</height>
              </rect>
             </property>
            </widget>
           </widget>
           <widget class="QSpinBox" name="spinBox_recentalerttimeout">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>460</y>
              <width>81</width>
              <height>36</height>
             </rect>
            </property>
            <property name="suffix">
             <string>min</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>470</y>
              <width>151</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>Recent alert timeout:</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="zabout_tab">
           <attribute name="title">
            <string>About</string>
           </attribute>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>40</y>
              <width>361</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>35</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>IntelPy</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>130</y>
              <width>141</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>by Swizzles Saissore</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>146</x>
              <y>100</y>
              <width>71</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>v0.9b</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>170</y>
              <width>161</width>
              <height>111</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="images.qrc">:/piebig/goodpie2.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="eve_statusbar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditHome_System</tabstop>
  <tabstop>pushButtonSet_Home</tabstop>
  <tabstop>horizontalSlider_AlertJumps</tabstop>
  <tabstop>clearLogButton</tabstop>
  <tabstop>lineEditEve_Log_Location</tabstop>
  <tabstop>pushButtonChoose_Eve_Log_Location</tabstop>
  <tabstop>lineEdit_alarm</tabstop>
  <tabstop>pushButtonChoose_alarm_Location</tabstop>
  <tabstop>checkbox_displayalerts</tabstop>
  <tabstop>checkBox_displayclear</tabstop>
  <tabstop>checkBox_displayall</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
